frontend/src/components/footer.tsx

```diff
@@ -15,28 +17,78 @@ import RiceLogoURL from '../images/rice-logo.png'
 import OpenStaxURL from '../images/openstax-logo.png'
 
 const Funders = () => {
+    const isMobile = useMediaQuery('(max-width: 768px)');
+    const isTablet = useMediaQuery('(max-width: 1024px)');
+
     return (
-        <Container bg='white'>
-            <Stack>
-                <Title order={4} className="fw-bold">Support from Scientific Agencies</Title>
-                <Flex justify='center' align='center' mx='auto'>
-                    <Anchor target="_blank" href="https://ies.ed.gov/">
-                        <Image alt="Institute of Education Sciences logo" src={IESLogoURL}/>
+        <Container bg="white" px={isMobile ? 16 : 32} pt={16} pb={40}>
+            <Stack gap={isMobile ? 16 : 24}>
+                <Title
+                    order={2}
+                    w={700}
+                    style={{
+                        fontFamily: 'Helvetica Neue',
+                        fontSize: isMobile
+                            ? '24px'
+                            : isTablet
+                                ? '30px'
+                                : '36px',
+                        lineHeight: isMobile
+                            ? '28px'
+                            : isTablet
+                                ? '36px'
+                                : '42px',
+                        letterSpacing: '-0.04em',
+                        textAlign: 'left',
+                    }}
+                >
+                    Support from scientific agencies
+                </Title>
+                <Flex
+                    direction={isMobile ? 'column' : 'row'}
+                    justify={isMobile ? 'flex-start' : 'space-between'}
+                    align={isMobile ? 'center' : 'flex-start'}
+                    gap={isMobile ? '24px' : '16px'}
+                >
+                    <Anchor
+                        target="_blank"
+                        href="https://www.nsf.org/gb/en"
+                        pl={isMobile ? 0 : isTablet ? 75 : 150}
```

Comment:
Is there a cleaner way to organize this? using flex properties above perhaps? hardcoding these paddings based on 3 different screen sizes is a little rough i think